code-challenges.test.js

```diff
@@ -53,7 +86,30 @@ const studyMinutesWeek2 = {
 
 // b) Create the function that makes the test pass.
 
+//Jest Testing:
+// describe("leastToGreatest", () => {
+//   it("Returns an array with the values sorted from least to greatest study minutes", () =>{
+//     expect(leastToGreatest(studyMinutesWeek1)).toEqual([15, 15, 20, 30, 30, 60, 90])
+//     expect(leastToGreatest(studyMinutesWeek2)).toEqual([10, 15, 20, 45, 60, 65, 100])
+//   })
+// })
+
 // Pseudo code:
+//Function Name: leastToGreatest
+//Input: Array with objects/key: value
+//Output: An array with JUST the values and it being ordered from small number to big number. 
+//Process: .sort() to organize each number from lowest to greatest. Object.value() to get the value of the object that is being given. Use the function (a,b) => a-b which will sort the numbers in ascending order even if there is a (100) triple digit number. 
+
+// const leastToGreatest = (object) => {
+//   const var1 = Object.values(object)
+//   Holds each array as: no key just value
+//   console.log(var1)
+//   const var2 = var1.sort((a,b) => a-b)
+//   return var2
+// }
+//Code sorts the numbers, but only issue is that 100 is on the second index instead of being last. 
```

Comment:
Seems like you figured this one out! `.sort` on its own will sort things based on their "Unicode code point values"...basically alphabetically based on the first letter of the string. This is why when dealing with numbers, it behaves weirdly. When we use the comparison function like you did above, it performs math on the numbers and determines which number is bigger, and moves the bigger number to the right and the smaller number to the left, which results in a numerical sort.